providers/edgedns/edgeDnsService.go

```diff
@@ -0,0 +1,305 @@
+package edgedns
+
+/*
+For information about Akamai's "Edge DNS Zone Management API", see:
+https://developer.akamai.com/api/cloud_security/edge_dns_zone_management/v2.html
+
+For information about "AkamaiOPEN-edgegrid-golang" library, see:
+https://github.com/akamai/AkamaiOPEN-edgegrid-golang
+*/
+
+import (
+	"fmt"
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/printer"
+	dnsv2 "github.com/akamai/AkamaiOPEN-edgegrid-golang/configdns-v2"
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid"
+)
+
+// EDInitialize initializes the "Akamai OPEN EdgeGrid" library
+func EDInitialize(clientSecret string, host string, accessToken string, clientToken string) {
+
+	eg := edgegrid.Config{
+		ClientSecret: clientSecret,
+		Host:         host,
+		AccessToken:  accessToken,
+		ClientToken:  clientToken,
+		MaxBody:      131072,
+		Debug:        false,
+	}
+	dnsv2.Init(eg)
+}
+
+// EDZoneDoesExist returns true if the zone exists, false otherwise.
+func EDZoneDoesExist(zonename string) bool {
+	_, err := dnsv2.GetZone(zonename)
+	return err == nil
+}
+
+// EDCreateZone create a new zone and creates SOA and NS records for the zone.
+// Akamai assigns a unique set of authoritative nameservers for each contract. These authorities should be
+// used as the NS records on all zones belonging to this contract.
+func EDCreateZone(zonename string, contractID string, groupID string) error {
+	zone := &dnsv2.ZoneCreate{
+		Zone:                  zonename,
+		Type:                  "PRIMARY",
+		Comment:               "This zone created by DNSControl (https://stackexchange.github.io/dnscontrol/)",
```

Comment:
Please use the url `http://dnscontrol.org` instead.

Reply:
Done.